lychee-bin/tests/cli.rs

```diff
@@ -606,6 +606,80 @@ mod cli {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn test_lycheecache_file() -> Result<()> {
+        let mut cmd = main_command();
+
+        let test_base_path = fixtures_path().join("cache");
+        assert!(
+            fs::metadata(test_base_path.join(".lycheecache")).is_err(),
+            ".lycheecache file should not exist before this test"
+        );
+
+        let mock_server_ok = mock_server!(StatusCode::OK);
+        let mock_server_err = mock_server!(StatusCode::NOT_FOUND);
+        let mock_server_exclude = mock_server!(StatusCode::OK);
+
+        let dir = tempfile::tempdir()?;
+        let mut file = File::create(dir.path().join("c.md"))?;
+
+        writeln!(file, "{}", mock_server_ok.uri().as_str())?;
+        writeln!(file, "{}", mock_server_err.uri().as_str())?;
+        writeln!(file, "{}", mock_server_exclude.uri().as_str())?;
+
+        // run first without cache to generate the .lycheecache file
+        cmd.current_dir(&test_base_path)
+            .arg(dir.path().join("c.md"))
+            .arg("--verbose")
+            .arg("--cache")
+            .arg("--exclude")
+            .arg(mock_server_exclude.uri())
```

Comment:
Realized you're constructing this command with the same arguments below.
I think you could construct it once and assign it to a variable?

```rust
let cache_cmd = cmd
        .current_dir(&test_base_path)
        .arg(dir.path().join("c.md"))
        .arg("--verbose")
        .arg("--cache")
        .arg("--exclude")
        .arg(mock_server_exclude.uri());
}

// and then
// cache_cmd.assert().stderr(...);
```

Reply:
done!